content/Registration and Authentication in your Django app with dj-rest-auth part1- Bruno Michetti.md

```diff
@@ -0,0 +1,215 @@
+## Registration and Authentication in your Django app with dj-rest-auth (part 1)
+
+A huge amount of existing applications have registration and authentication for users. Maybe every developer in the world has implemented something related to this in their work or while they learned. After the creation of the [Django REST framework](https://www.django-rest-framework.org/), Django developers started to implement more and more app-level REST API endpoints.  
```

Comment:
Maybe we can rephrase "in their work or while they learned.", what do you think?

Reply:
`at work or while learning` WDYT?